python/xorbits/core/utils/docstring.py

```diff
@@ -62,19 +63,27 @@ def add_docstring_disclaimer(
     if base_indentation is None:
         return doc
 
+    warning_msg = (
+        f"\n\n{base_indentation}.. warning:: This method has not been implemented yet. Xorbits will try to "
+        f"execute it with pandas. "
```

Comment:
`xorbits.numpy` also use this method to generated docstrings, so better replace `pandas` with `docstring_src_module` and properly handle the case when `docstring_src_module` being `None`.